Gaijinizer patch/Armors.txt

```diff
@@ -1001,29 +1001,29 @@ Cloth Clothes#StrateUnsure
 > BEGIN STRING
 マリオンの世界に自生している赤リンゴ
 \\C[5]マリオンの回復スキルの対象が味方全体になる。
-> CONTEXT: Armors/101/description
-
+> CONTEXT: Armors/101/description < UNTRANSLATED
+ADD STUFF HERE
 > END STRING
```

Comment:
There's a few entries in this file that cover descriptions for the Apples you'll need to cover.